docs/how-tos/configure-teleop-workspace.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "Configure a teleop workspace"
+linkTitle: "Configure a teleop workspace"
+weight: 10
+type: "docs"
+description: "Create and configure a teleop workspace with widgets."
+images: ["/how-tos/teleop/full-workspace.png"]
+icon: true
+tags: ["teleop", "configuration"]
+languages: []
+viamresources: ["sensor", "camera", "movement sensor"]
+platformarea: ["viz", "data"]
+level: "Intermediate"
+date: "2024-11-13"
+# updated: "2024-08-26"  # When the tutorial was last entirely checked
+cost: "0"
+---
+
+You can use teleop to create a custom workspace where you can visualize data from a machine.
+Follow this guide to learn how to create a custom workspace using teleop, visualizing data from a camera, a sensor, or a movement sensor.
+
+{{% alert title="In this page" color="info" %}}
+
+- [Configure a workspace](#configure-a-workspace)
+
+{{% /alert %}}
+
+## Prerequisites
+
+{{% expand "A configured machine with teleoperable components" %}}
+
+Make sure your machine has at least one of the following:
+
+- A [camera](/components/camera/)
+- A [movement sensor](/components/movement-sensor/)
+- A [sensor](/components/sensor/)
+
+See [configure a machine](/how-tos/configure/) for more information.
+
+{{% /expand%}}
+
+## Configure a workspace
+
+{{< table >}}
+{{% tablestep %}}
+**1. Create a workspace in the Viam app**
+
+Log in to the [Viam app](https://app.viam.com/).
+
+Navigate to the **FLEET** page's **TELEOP** tab.
+Create a workspace by clicking **+ Create workspace**.
+Give it a name.
+
+{{<imgproc src="/how-tos/teleop/blank-workspace.png" resize="800x" style="width: 500px" class="fill aligncenter" declaredimensions=true alt="Blank teleop page.">}}
+
+{{% /tablestep %}}
+{{% tablestep %}}
+**2. Add widgets**
+
+You are prompted to add a widget to add teleop visualizations to your workspace.
+Click **Add widget** and select the appropriate widget for your machine.
+Repeat as many times as necessary.
+
+- If you have a camera, select **Camera stream**, which allows you to view a camera stream.
+  Enter the name of your camera.
+
+- If you have a movement sensor, select a **GPS map**, which allows you to map the location of a machine.
+  Enter the name of your movement sensor.
+  Toggle **Historic positions** to on to see how position has changed over time.
+
+- If you have a sensor, select **Stat** to view a single sensor value.
+  Enter the name of the sensor for **Sensor name** and enter the key for the value in **Path**, for example `time_unix`.
```

Comment:
I think for these it would be good to add a tip for people trying to figure out how to get the path (go to machine's test panel and check the returned json object. should probably also include what to do if it's inside another object)

However, this should ideally be a tooltip in the app. Please file the appropriate ticket with a suggestion for copy.

Reply:
They are going to make it autoselect so I think it's not worth filing that as it would quickly be deprecated, does that make sense? @npentrel

Reply:
sounds good